### PR TITLE
src: update all services to use API.Node class

### PR DESCRIPTION
**Pull request by gctucker**
Update all the pipeline services to use the new API.Node bindings class for all node-related operations.

### CODE REVIEW

**Comment by gctucker** [comment]
This PR is conflicting with other earlier PRs so it's not being deployed on staging.  Please let me know if they get merged and when it needs to be rebased.

**Comment by nuclearcat** [comment]
Can you rebase? I merged some of PRs that qualify to be merged

File: src/trigger.py [hunk 1]
```diff
@@ -70,7 +69,7 @@ def _run_trigger(self, build_config, force, timeout):
             'timeout': checkout_timeout.isoformat(),
         }
         try:
-            self._api.create_node(node)
+            self._api.node.create(node)
```
[the remark this answers is not in the document]
Comment (gctucker) [reply src/trigger.py]: Ah yes, sorry.  I've pushed a fix.

**Comment by nuclearcat** [comment]
```
    today at 6:28:43 PMTraceback (most recent call last):
    today at 6:28:43 PM  File "/home/kernelci/./pipeline/trigger.py", line 160, in <module>
    today at 6:28:43 PM    status = opts.command(configs, opts)
    today at 6:28:43 PM             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    today at 6:28:43 PM  File "/home/kernelci/./pipeline/trigger.py", line 153, in __call__
    today at 6:28:43 PM    return Trigger(configs, args).run(args)
    today at 6:28:43 PM           ^^^^^^^^^^^^^^^^^^^^^^
    today at 6:28:43 PM  File "/home/kernelci/./pipeline/trigger.py", line 31, in __init__
    today at 6:28:43 PM    self._current_user = self._api.whoami()
    today at 6:28:43 PM                         ^^^^^^^^^^^^^^^^
    today at 6:28:43 PMAttributeError: 'LatestAPI' object has no attribute 'whoami'
    today at 6:30:50 PMContainer stopped
    ```

**Review by JenySadadia — APPROVED** [review]
Staging looks good to me along with https://github.com/kernelci/kernelci-pipeline/pull/373.

**Review by nuclearcat — APPROVED** [review]
Tested on staging

**Pull request merged**